### ISSUE TITLE
Accessibility - Important information will be missed by a screen reader in Forms Mode

### ISSUE THREAD

**Issue by ralliartse** [DependencyTrack/frontend#1614]
### Current Behavior

In the create project form, when a screen reader is focused on any text box (project name, version, classifier, team, etc), it doesn't read the text in the information box beside.

Non visual users might miss important information when trying to create new projects.

<img width="1006" height="457" alt="Image" src="https://github.com/user-attachments/assets/96f12246-aaf1-4473-837f-90e0a0d0495a" />

### Proposed Behavior

When the non visual user tab through the form, the screen reader should tell this user the information included in these information box.

### Checklist

- [x] I have read and understand the [contributing guidelines](https://github.com/DependencyTrack/dependency-track/blob/main/CONTRIBUTING.md#filing-issues)
- [x] I have checked the [existing issues](https://github.com/DependencyTrack/frontend/issues) for whether this enhancement was already requested
